src/javascript/app/components/date_picker.js

```diff
@@ -115,7 +115,7 @@ const DatePicker = (() => {
 
     const formatDate = (date, add) => padLeft(date + (add || 0), 2, '0');
 
-    const toDate = date => [date.getFullYear(), formatDate(date.getMonth(), 1), formatDate(date.getDate())].join('-');
+    // const toDate = date => [date.getFullYear(), formatDate(date.getMonth(), 1), formatDate(date.getDate())].join('-');
```

Comment:
Do you need to keep this comment inside the code?
If yes, please add a 'TODO' and explain why

Reply:
it's deleted.